### DISCUSSION TITLE
Project Source Code V 3.7.17

### DISCUSSION THREAD

**Discussion by Mutato115** [ChestShop-authors/ChestShop-3#408] [General]
Is it possible to get the source code for ChestShop Version 3.7.17?
I need to get a build that works with Bukkit for Minecraft 1.7.10 and extracting & decompiling the JAR isn't the solution I want...

**Comment by Phoenix616** [comment]
The full source code with all of its history is in the repository ;) Just find whatever commit which updated it to 1.8 and use the one before that. Unfortunately there weren't any tags for a while so you are going to have to either look at the blame of the pom or use the release data from dev bukkit.
